src/content/release-notes.mdx

```diff
@@ -5,6 +5,42 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.19.0
+
+16 April 2024
+
+This versions is compatible with Kubernetes versions 1.25 to 1.29
+
+### New Features
+
+- Okteto Insights: Pod metrics v2  <!-- 7897, 7906 -->
+- Okteto Insights: User metrics  <!-- 7898 -->
+- Okteto Insights: Namespace and Preview metrics  <!-- 7900 -->
+
+### Improvements
+
+- Okteto CLI: `okteto deploy` will now ignore host volumes. To manage the mounting of volumes, use a Dockerfile with the `COPY` instruction instead of references in your Docker compose.
```

Comment:
This is a breaking change. We should mark it as a breaking change and link to the community post about how to overcome the breaking change

Reply:
Followed breaking changes convention and made a section for this. Leaving this open until the community post goes live